easybuild/easyconfigs/b/Bazel/Bazel-6.5.0-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,31 @@
+name = 'Bazel'
+version = '6.5.0'
+
+homepage = 'https://bazel.io/'
+description = """Bazel is a build tool that builds code quickly and reliably.
+It is used to build the majority of Google's software."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+source_urls = ['https://github.com/bazelbuild/%(namelower)s/releases/download/%(version)s']
+sources = ['%(namelower)s-%(version)s-dist.zip']
+patches = ['Bazel-6.5.0_py3.12_pytest_assertEqual.patch']
+checksums = [
+    {'bazel-6.5.0-dist.zip': 'fc89da919415289f29e4ff18a5e01270ece9a6fe83cb60967218bac4a3bb3ed2'},
+    {'Bazel-6.5.0_py3.12_pytest_assertEqual.patch': '2670dd5c393970ba20db2c98cf0208df7190ff339ccb66fee9a6d48aaaf3ede6'},
+]
+
+builddependencies = [
+    ('binutils', '2.42'),
+    ('Python', '3.12.3'),
+    ('Zip', '3.0'),
+]
+
+dependencies = [
+    ('Java', '11.0.20', '', SYSTEM),
```

Comment:
Can we use Java/21 here?

Reply:
@boegel Build of Bazel 6.5.0 fails for both, Java/21 and Java/21.0.2.